Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8.3 as builder
+WORKDIR /go/src/tiedot/
+COPY ./    .
+WORKDIR /go/
+RUN go get -v -d tiedot
```

Comment:
If this is to get the `tiedot` dependencies, it may be better to not change the workdir to `/go/` and instead run `go get -v -u -d .` from within `/go/src/github.com/HouzuoGuo/tiedot`.